sale_order_invoicing_queued/models/sale_order.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+try:
+    from odoo.addons.queue_job.job import job
+except ImportError:
+    import logging
+    import functools
+
+    logging.getLogger().debug("Can't `import queue_job`.")
```

Comment:
```suggestion
    logging.getLogger(__name__).warning("Can't `import queue_job`.")
```

Reply:
It mustn't be warning: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#imports